lms/templates/footer-extra.html

```diff
@@ -0,0 +1,12 @@
+<%page args="theme_folder" />
+## mako
+
+<%namespace name='static' file='/static_content.html'/>
+
+<!-- Include jQuery -->
+##<script src="${static.url('js/jquery-1.11.2.min.js')}"></script>
```

Comment:
Optional: While we're here, can we just remove this instead of commenting it out? (And remove the `jquery-1.11.2.min.js` file, and update the comment on line 6 to say "Include jQuery mobile" only)